frontend/scripts/react-components/profile-search/profile-search.reducer.js

```diff
@@ -0,0 +1,24 @@
+import clone from 'lodash/clone';
+import { createReducer } from 'store';
+import {
+  SET_PROFILE_SEARCH_ERROR_MESSAGE,
+  SET_PROFILE_SEARCH_NODES
+} from 'react-components/profile-search/profile-search.actions';
+
+const initialState = {
+  nodes: [],
+  errorMessage: null
+};
+
+const homeReducer = {
+  [SET_PROFILE_SEARCH_NODES](state, action) {
+    const { nodes } = action.payload;
+    return { ...state, nodes: clone(nodes) };
```

Comment:
In the `homeReducer` I use clone because the same reducer saves arrays and objects, and this way I don't have to worry about what I'm receiving. In this case, it doesn't really matter, but we could trim a (perhaps negligible) dependency using `[...nodes]` syntax.

Reply:
fixed, thanks for the tip :+1: